pyninjasphere/services/HTTP/http_client.py

```diff
@@ -0,0 +1,54 @@
+#created by: Wilko
+#Project team: TechMasters
+
+import urllib.request
+
+
+class HTTPClient:
+    """The HTTPClient sends and receives data over HTTP"""
+
+    def __init__(self, headers = None):
+        self.values = None
+        self.headers = headers
+        if headers is None:
+            self.headers = {
+                'Content-Type': 'application/json',
+            }
+
+    def get_data_with_http_request(self, url):
+        """
+        Gets data from an url
+        :param url: http url
+        :return: the requested data
+        """
+        print("[SERVICES][HTTPCLIENT] sendHTTPRequest")
+        request = self.create_request(url)
+        response = self.create_response(request)
+        data = self.decode_response(response)
+        return data
+
+    def create_request(self, url):
+        """
+        Creates a request object
+        :param url: http url
+        :return: A request object
+        """
+        return urllib.request.Request(url)
+
+    @staticmethod
+    def create_response(request):
```

Comment:
Deze naam suggereert een simpele object creation of zo, maar hier wordt daadwerkelijk de http call gedaan. Ik zou de methode navent noemen.